src/test/run-cli-tests

```diff
@@ -41,7 +41,16 @@ mkdir -p "$FAKE_HOME"
 # cram doesn't like seeing the same foo.t basename twice on the same
 # run, so run it once per directory
 FAILED=0
+FAILEDTOOLS=""
 for tool in "$SRCDIR"/cli/*; do
+    if [ x`uname`x = xFreeBSDx ]; then
+    	if [ x"$tool"x = x"$SRCDIR/cli/rbd"x ]; then
```

Comment:
Skipping this for the time is rather essential, since otherwise the Makefiles do not continue to the next set of tests. And then it it not possible to get a decent coverage when we Jenkins-ify the building. And I haven't looked into it, but I guess that adopting the tests for FreeBSD is rather big by itself.